vaadin-radio-button.html

```diff
@@ -17,33 +17,169 @@
 -->
 
 <link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../polymer/lib/mixins/gesture-event-listeners.html">
+<link rel="import" href="../vaadin-themable-mixin/vaadin-themable-mixin.html">
+<link rel="import" href="../vaadin-control-state-mixin/vaadin-control-state-mixin.html">
 
-<dom-module id="vaadin-radio-button">
+<dom-module id="vaadin-radio-default-theme" theme-for="vaadin-radio-button">
   <template>
     <style>
       :host {
         display: inline-block;
+        outline: none;
+      }
+
+      [part="wrapper"] {
+        display: inline-flex;
+        align-items: center;
+        outline: none;
+      }
+
+      :host([focus-ring]) {
+        box-shadow: 0 0 2px 2px Highlight;
+      }
+
+      [part="native-radio"] {
+        margin-right: 0;
+      }
+
+      [part="label"] {
+        padding: 0 5px;
+        outline: none;
+      }
+
+      :host([disabled]) [part="label"] {
+        opacity: 0.5;
       }
     </style>
-    <slot></slot>
+  </template>
+</dom-module>
+
+<dom-module id="vaadin-radio-button">
+  <template>
+
+    <label part="wrapper" id="label">
+      <input
+        id="nativeRadio"
+        type="radio"
+        part="native-radio"
+        checked="{{checked::change}}"
+        disabled$="[[disabled]]"
+        role="presentation"
+        tabindex="-1">
+
+      <span part="radio"></span>
+
+      <span part="label">
+        <slot></slot>
+      </span>
+    </label>
+
   </template>
 
   <script>
     if (!Polymer.Element) {
-      throw new Error(`Unexpected Polymer version ${Polymer.version} is used, expected v2.0.0 or later.`);
+      throw new Error('Unexpected Polymer version ${Polymer.version} is used, expected v2.0.0 or later.');
     }
 
-    class VaadinRadioButton extends Polymer.Element {
-      static get is() {
-        return 'vaadin-radio-button';
-      }
+    (function() {
```

Comment:
Is there a consistent agreement about IIFE usage? AFAIK most of Vaadin elements don't use it.